doc/classes/CubeMap.xml

```diff
@@ -19,6 +21,7 @@
 			<return type="int">
 			</return>
 			<description>
+				Return the height of the CubeMap.
```

Comment:
- Return -> Returns
- Per the writing guidelines, use possessive:

Returns the CubeMap's height.